jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageReference.java

```diff
@@ -86,6 +86,7 @@ public static ImageReference parse(String reference) throws InvalidImageReferenc
     String tag = matcher.group(3);
     String digest = matcher.group(4);
 
+    // If no registry was matched, use Docker Hub by default.
```

Comment:
I wonder if since we're behaving a certain way only if `DEFAULT_REGISTRY` is docker hub that we should just rename `DEFAULT_REGISTRY` to indicate that it is actually dockerhub: like `DOCKER_HUB` or something?

Reply:
Also since changing default without looking down at this section of code would break it, since `library` is specific to dockerhub?